charts/local-kubevirt/update.bash

```diff
@@ -56,11 +56,15 @@ wget https://github.com/kubevirt/containerized-data-importer/releases/download/$
 wget https://github.com/kubevirt/containerized-data-importer/releases/download/$latest_cdi_release/cdi-cr.yaml -O ${dir}/templates/cdi-cr.yaml
 
 pushd $dir/templates
+sed -i -e "s/\(^[ ]*namespace:\) cdi$/\1 kubevirt/" cdi-operator.yaml
```

Comment:
ensuring CDI is deployed in kubevirt namespace

Reply:
How about you put this, and maybe more of your nice comments here on Github, into code comments in the scripts? :-)

Also please rename this to use .sh instead of .bash, or is there a super strong reason to make this our only script in the entire repo to require .bash file extension?

Reply:
Thank you, solid points!

> or is there a super strong reason to make this our only script in the entire repo to require .bash file extension?

no good reason, just poor old habits die hard...